### PR TITLE
Remove CHAMPVA IVC code from Simple Forms API: Part I

### PR DESCRIPTION
**Pull request by Thrillberg**
## Summary
This PR removes CHAMPVA IVC forms code from the Simple Forms API module. CHAMPVA has already moved over to their own module so this should all be dead code.

[Part II is here.](https://github.com/department-of-veterans-affairs/vets-api/pull/16410)

## Related issue(s)
https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team/80840

### CODE REVIEW

**Review by balexandr — APPROVED** [review]
@Thrillberg this looks to be everything. Nothing is jumping out missing.

**Review by pennja — APPROVED** [review]
Looks good, great work! 👍

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -956,6 +956,7 @@ modules/dhp_connected_devices @department-of-veterans-affairs/digital-health-pla
 modules/facilities_api @department-of-veterans-affairs/vfs-facilities-frontend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 modules/health_quest @department-of-veterans-affairs/vsa-healthcare-health-quest-1-backend
 modules/income_limits @department-of-veterans-affairs/vfs-public-websites-frontend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+modules/ivc_champva @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (balexandr) [inline .github/CODEOWNERS:959]: I'm actually adding this `modules/ivc_champva @department-of-veterans-affairs/champva-engineering @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group` in another PR

File: modules/ivc_champva/app/services/ivc_champva/pdf_filler.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def generate(current_loa = nil)
     end
 
     def mapped_data
-      template = Rails.root.join('modules', 'simple_forms_api', 'app', 'form_mappings', "#{form_number}.json.erb").read
+      template = Rails.root.join('modules', 'ivc_champva', 'app', 'form_mappings', "#{form_number}.json.erb").read
       b = binding
```
Comment (balexandr) [inline modules/ivc_champva/app/services/ivc_champva/pdf_filler.rb:36]: good catch!

File: .github/CODEOWNERS (hunk 1 above)
Comment (balexandr) [reply .github/CODEOWNERS:959]: If you can get this PR merged before mine go with it but you'll have to update the team

**Review by pennja — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**